services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx

```diff
@@ -62,6 +64,8 @@ export const EnvironmentCard: React.FC<{ environment: string }> = (props) => {
 
 export const EnvironmentGroupCard: React.FC<{ environmentGroup: EnvironmentGroup }> = (props) => {
     const { environmentGroup } = props;
+    // all envs in the same group have the same priority
+    const priorityClassName = useEnvironmentPriorityClassName(environmentGroup.environments[0]?.name);
```

Comment:
This does not seem completely right. If the first element in `environmentGroup.environments` has a lower priority but the next element has a higher priority, then the priority of the envGroup will be lower than I expected. Shouldn't we look into every environment in the environment group and have the `priorityClassName` be equal to the highest priority environment in the environment group?

Reply:
Or do we already sort the environments by priority?

Reply:
https://github.com/freiheit-com/kuberpult/pull/683